.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: An actionable title e.g. "Update the icon for..." or "Fix the behavior of..."
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
How about a link to this resource from Mozilla? https://bugzilla.mozilla.org/page.cgi?id=bug-writing.html